src/main/kotlin/com/cognifide/gradle/environment/hosts/HostUpdater.kt

```diff
@@ -86,4 +88,31 @@ class HostUpdater(val environment: EnvironmentExtension) {
             }
         }
     }
+
+    private fun execOnMacAndValidateResult(scriptFile: File) {
+        val errorOutput = ByteArrayOutputStream()
+        val execResult = project.exec {
+            it.errorOutput = errorOutput
+            it.isIgnoreExitValue = true
+            it.commandLine("sh", scriptFile.toString())
+        }
+        if (execResult.exitValue == 0) {
+            logger.lifecycle("Environment hosts successfully updated.")
+        } else {
+            val errorText = String(errorOutput.toByteArray())
+            if (errorText.contains("execution error: Error: Unable to access jarfile")) {
+                logger.error(
+                        """
+                    Failed to update environment hosts. Unable to access executable. You may have placed your source files under 
```

Comment:
how about same mechanics/message style as in https://github.com/Cognifide/gradle-environment-plugin/blob/master/src/main/kotlin/com/cognifide/gradle/environment/docker/Container.kt#L112